### ISSUE TITLE
Include translations right into the "compiled" cockpit.js

### ISSUE THREAD

**Issue by marusak** [cockpit-project/cockpit#13906]
Include translations right into the "compiled" cockpit.js, so that it works for third-party pages like podman as well and not bundle it into every po.lang.js file.

See #13901

**Comment by allisonkarlitskaya** [comment]
Shouldn't this just be as simple as the page in question including the base1 translation files as a script fragment?

Like we already have

```
    <script src="../*/po.js"></script>
```

at the top of our files.  Why not also:

```
    <script src="../../base1/*/po.js"></script>
```

or whatever?

`cockpit.locale()` itself already contains code to merge the translations, not replace them.

**Comment by martinpitt** [comment]
Yes, that sounds excellent! At this point I'm not even sure any more if we have some hacks to include/duplicate bridge's translations everywhere else.
